test/integration/shells_int_test.go

```diff
@@ -49,7 +48,7 @@ func (suite *ShellsIntegrationTestSuite) TestShells() {
 			}
 
 			// Run the checkout in a particular shell.
-			cp = ts.SpawnShellWithOpts(shell, e2e.OptAppendEnv(constants.DisableRuntime+"=true"))
+			cp = ts.SpawnShellWithOpts(shell)
```

Comment:
Set async runtime?

Reply:
Nope, we need it for the same reason as stated above.